eg6.py

```diff
@@ -0,0 +1,15 @@
+"""Find how many number of times the substring occuranceurs in the given string."""
+
+string = "PQRQRQRQQRQ"    #given string
```

Comment:
Use another name as variable, because string is a reserved keyword in the document. Please check the doc if you don't remember it.